### ISSUE TITLE
Prevent FixedForeground scaling

### ISSUE THREAD

**Issue by kenanchristian** [bamlab/react-native-image-header-scroll-view#46]
Currently, if the scroll view is at the top when user tries to scroll to the top again the image will scale with the FixedForeground element. Can I disable the scaling for the FixedForeground element? So only the background image is scaled

**Comment by kresent** [comment]
Yes, I stumbled upon same issue. But somehow the arrow in the Dr.Who example is not scaled but still fixed. Wonder how to keep the arrow in place and still have scaling background
